README.md

```diff
@@ -1,6 +1,6 @@
 # node-build
 
-[![Build Status](https://travis-ci.org/OiNutter/node-build.png?branch=master)](https://travis-ci.org/OiNutter/node-build)
+[![Build Status](https://travis-ci.org/pongstr/node-build.svg?branch=master)](https://travis-ci.org/pongstr/node-build)
```

Comment:
switching to svg is a good idea, but not sure why you changed all the links to your username. We need to keep the links pointing to the main repo (all below, too)